api/controllers/preregistrationController.js

```diff
@@ -302,6 +323,16 @@ exports.linkAesirX = async (req, res) => {
         dateAesirXAccount: new Date(),
       }
     );
+    const sendSlack = async () => {
+      try {
+        const resp = await axios.post(`${webhook}`, {
+          text: `Someone linked AesirX successfully`,
```

Comment:
Please tell the id and AesirX account who registered

Reply:
Please take a look again